<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPlanningDockWindow</class>
 <widget class="QDockWidget" name="QgsPlanningDockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>758</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>116</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>整治模式优选</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="styleSheet">
       <string notr="true">QTreeWidget::item{height:45px}</string>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>整治区片划分</string>
       </property>
       <item>
        <property name="text">
         <string>生态适宜性评价</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>生态适宜性分级</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>区片划分</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>整治方向选择</string>
       </property>
       <item>
        <property name="text">
         <string>村庄布点规划</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>整治模式选择</string>
       </property>
       <item>
        <property name="text">
         <string>模式选择</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>村庄模式优选</string>
       </property>
       <item>
        <property name="text">
         <string>村庄模式选择</string>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout1"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout4"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page5">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout5"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout6"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
